fuel/app/classes/controller/test.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class Controller_Test extends Controller
+{
+  public function action_ci($arg = NULL)
+  {
+    if ($arg !== NULL) {
```

Comment:
__[code_sniffer]__
* Tabs must be used to indent lines; spaces are not allowed
* Opening brace must be on next line from its condition
* Opening brace must have the same indentation than its condition
* Closing brace must be followed by EOL char
* TRUE, FALSE and NULL must be lowercase; expected "null" but found "NULL"